rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -234,18 +265,112 @@ For example, if you work in Buenos Aires (ART (UTC -3)), specify a schedule that
+
+You can set either the day of the week or day of the month, but not both. Replace either the day of the week or day of the month with an asterisk.
+
+The Scheduler supports most standard Cron expressions. Here are a few good and bad examples:
+
+[%header,cols="3*"]
+|===
+|Expression |Behavior |Explanation 
+|`5 4 3 2 1` | At 04:05 AM on every Monday in every February and (?) on the 3rd of every February| This Cron expression is invalid. You can set either the day of the week or day of the month, but not both. Replace either the day of the week or day of the month with an asterisk. If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week. See next two examples. 
+|`5 4 3 2 *` | At 04:05 AM on the 3rd of every February| If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week. You can't specify the year. 
+|`5 4 * 2 3` | At 04:05 AM on every Wednesday of every February| If you want to run a process exactly once on a specific time, you must specify the day of the month but not the day of the week.You can't specify the year. 
+|`*/7 * * * *` |Run at minute 00, 07, 14, 21, 28, 35, 42, 49, 56 of every hour (of every day of every month on every weekday). |Starts at minute 0 because it's the first of all values in the valid range (*).
+|`45/05 * * * *` |Run at minutes 45, 50, and 55 of every hour |
+|`* * * * *` |Run every minute forever |
+|`* 9-16 * * 1-5` |Every minute, between 09:00 AM and 04:59 PM, Monday through Friday |
+|`00,10,42 9-16 1-10,20-30 2-7,12 *`` |At 0, 10, and 42 minutes past the hour, between 09:00 AM and 04:59 PM, on day 1 through 10 and 20 through 30 of the month, and on Friday, only in February through July and December |You can combine lists and ranges.
+|===
+
+Non-existant times, such as 'missing hours' during daylight savings conversion, never match the bot's time-date setting, causing processes scheduled during the 'missing times' not to run. Similarly, times that occur more than once (again, during daylight savings conversion) cause matching processes to run twice.
+
+'''
+=====
++
+** *Calendar Scheduling*
 +
+Use this option to edit run configurations that already use calendar scheduling.
++ 
+Add one or more fixed schedules within which the productive run is carried out continuously. For each schedule, define time intervals within which the process is executed. RPA Manager combines all schedules to compute the execution times.
++
+.How does calendar scheduling work?
+[%collapsible]
+=====
+
+'''
+
+You can create multiple schedules. These may also overlap. All schedules together make up a time frame within which the process may be executed. The executing bot checks every minute whether it's within this time frame or not. A new process run isn't started until all conditions have been fulfilled.
```

Comment:
```suggestion
You can create multiple overlapping schedules. All schedules together make up a time frame within which the process executes. The executing bot checks every minute whether the process is within the time frame. A new process run doesn't start until all conditions are fulfilled.
```

Reply:
The process may be executed = It is allowed to run in this time frame. But might not be executed because of Bot Service Times or Application Down Times.